odoo/addons/base/ir/ir_cron.py

```diff
@@ -218,8 +226,21 @@ def _acquire_job(cls, db_name):
                 # we're exiting due to an exception while acquiring the lock
                 lock_cr.close()
 
+        try:
+            with db.cursor() as cr:
+                cr.execute("SELECT MIN(nextcall) as nextcall FROM ir_cron WHERE active = 't'")
```

Comment:
nitpicking: I like `"WHERE active"` better it does not require the reader to know the valid representations for "true". But that's just me ;-)